CHANGELOG.md

```
@@ -1,5 +1,9 @@
 ## 1.3.1
 
+* Fix build errors with flutter 3.35.0 or superior
+
+## 1.3.1
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)